client/components/site-verticals-suggestion-search/popular-topics.jsx

```diff
@@ -30,22 +30,22 @@ const POPULAR_TOPICS = {
 		translate( 'Restaurants' ),
 	],
 	blog: [
-		translate( 'Travel' ),
-		translate( 'Fashion' ),
-		translate( 'Education' ),
 		translate( 'Food' ),
-		translate( 'Music' ),
+		translate( 'Travel' ),
+		translate( 'Film' ),
 		translate( 'Photography' ),
-		translate( 'Sports' ),
+		translate( 'Local' ),
+		translate( 'People' ),
+		translate( 'Sport' ),
 	],
 	professional: [
-		translate( 'Education' ),
-		translate( 'Higher Education & Academy' ),
-		translate( 'Travel' ),
-		translate( 'School' ),
-		translate( 'Website Designer' ),
-		translate( 'Fashion' ),
-		translate( 'Portfolio' ),
+		translate( 'Photographer' ),
+		translate( 'Web Designer' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Web designer' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).